packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -6,6 +6,52 @@ import { Setup } from "test/setup/Setup.sol";
 import { Events } from "test/setup/Events.sol";
 import { FFIInterface } from "test/setup/FFIInterface.sol";
 import "scripts/DeployConfig.s.sol";
+import { ForgeArtifacts } from "scripts/ForgeArtifacts.sol";
+import { Executables } from "scripts/Executables.sol";
+import { Vm } from "forge-std/Vm.sol";
+
+struct AbiEntry {
+    string fnName;
+    bytes4 sel;
+}
+
+struct Abi {
+    string contractName;
+    AbiEntry[] entries;
+}
+
+/// @dev Returns the function ABIs of all L1 contracts.
+function getL1ContractFunctionAbis() returns (Abi[] memory abis_) {
```

Comment:
Can you put these in the forge library at `scripts/ForgeArtifacts.sol`? It already has similar functions in there

Reply:
sure

Reply:
OK, done.